drydock/templates/drydock/k8s/ingress/extra-hosts.yml

```diff
@@ -0,0 +1,37 @@
+{% for host in DRYDOCK_INGRESS_EXTRA_HOSTS -%}
+{% if host not in [NOTES_HOST, MINIO_HOST, MINIO_CONSOLE_HOST, MFE_HOST] -%}
```

Comment:
Could we handle `MINIO_HOST` and `MINIO_CONSOLE_HOST` as extra hosts instead of having a dedicated ingress file for MinIO?